packages/core/.depcheckrc.yml

```diff
@@ -13,3 +13,4 @@ ignores:
   - 'one'
   # types
   - '@types/json-stable-stringify'
+  - '@types/merge-deep'
```

Comment:
why add it btw when we're not explicitly using it?

Reply:
ah nvm we're implicitly providing types when requiring (in couple places)

https://github.com/LavaMoat/LavaMoat/blob/f27bf4354c561f66a039921bb2ac6abb1a625819/packages/core/src/mergePolicy.js#L6

https://github.com/LavaMoat/LavaMoat/blob/f27bf4354c561f66a039921bb2ac6abb1a625819/packages/core/test/util.js#L7

Reply:
yeah, we're explicitly using `@types/merge-deep` because we have a `require('merge-deep')` in a typechecked file.  unfortunately, I don't see how this depcheck tool could know that, if in the future, we _stop_ using `merge-deep`, then we should also drop `@types/merge-deep`.

Reply:
[dependency-cruiser](https://www.npmjs.com/package/dependency-cruiser) might be a more suitable alternative.  I haven't gotten very deep into it though.